### PR TITLE
Update Status function changes for `QueueStore`

### PR DESCRIPTION
**Pull request by aneeshsharma**
Made these updates to the `QueueStore` since now we are using `bool` values for the paused and delete status
- Changed `UpdateQueueStatus` to 2 functions - `UpdateQueuePauseStatus` and `UpdateQueueDeleteStatus`
- Changed `UpdateTokenStatus` to `UpdateTokenDeleteStatus`

### CODE REVIEW

**Review by maaverik — APPROVED** [review]

File: internal/persistence/persistence.go [hunk 1]
```diff
@@ -7,12 +7,15 @@ import (
...
 	// Read a queue by id.
 	ReadQueue(models.QueueId) models.Queue
 
-	// Set the queue status to new value.
-	UpdateQueueStatus(models.QueueId, models.QueueStatus)
+	// Set the queue pause status to new value.
+	UpdateQueuePauseStatus(models.QueueId, bool)
+
+	// Set the queue delete status to new value.
+	UpdateQueueDeleteStatus(models.QueueId, bool)
```
Comment (daltonfury42) [inline internal/persistence/persistence.go]: Let's rather have three functions

PauseQueue(models.QueueId)
ResumeQueue(models.QueueId)
DeleteQueue(models.QueueId)

File: internal/models/queue.go [hunk 2]
```diff
@@ -8,19 +8,19 @@ import (
...
-	// Name of the queue. The name is used in generating queue urls, for ex 
-	// https://simplq.me/j/<QueueName>. The storage layer guarantees sure that 
+	// Name of the queue. The name is used in generating queue urls, for ex
+	// https://simplq.me/j/<QueueName>. The storage layer guarantees sure that
 	// there is only one queue by a given name.
 	QueueName string
 
 	// Set to true if the queue is temporarily not issuing tokens
-	bool IsPaused
+	IsPaused bool
```
Comment (daltonfury42) [inline internal/models/queue.go:20]: Oh. I had compiled it before checkin, and it worked because the file was never imported anywhere I think.

**Review by daltonfury42 — CHANGES_REQUESTED** [review]

File: internal/models/queue.go (hunk 2 above)
Comment (aneeshsharma) [reply internal/models/queue.go:20]: Yeah

**Comment by aneeshsharma** [comment]
I have made the changes @daltonfury42

File: internal/persistence/persistence.go [hunk 3]
```diff
@@ -21,5 +27,5 @@ type QueueStore interface {
 	ReadToken(models.TokenId)
 
 	// Set token status to new value.
-	UpdateTokenStatus(models.TokenId, models.TokenStatus)
-}
\ No newline at end of file
+	UpdateTokenDeleteStatus(models.TokenId, bool)
```
Comment (daltonfury42) [inline internal/persistence/persistence.go]: Let us call this as `RemoveToken(models.TokenId)`

**Review by daltonfury42 — APPROVED** [review]

**Comment by aneeshsharma** [comment]
I'll merge this PR

**Pull request merged**